sdk/xmake.lua

```diff
@@ -232,6 +232,11 @@ rule("firmware")
 			add_defines("SIMULATION")
 		end
 
+		if board.stack_high_water_mark then
```

Comment:
At least the Sail json should probably be updated to include this.

Reply:
OK, once the Sail PR is merged I'll do that here. I need to update it to say high water mark instead of low water mark.